FlowCrypt/Controllers/Settings/Settings List/SettingsViewController.swift

```diff
@@ -31,17 +31,35 @@ final class SettingsViewController: TableNodeViewController {
             case .experimental: return "settings_screen_experimental".localized
             }
         }
+
+        static func allCases(with rules: OrganisationalRules?) -> [Settings] {
+            guard let rules = rules else {
+                return allCases
+            }
+            let cases: [Settings]
+            if !rules.canBackupKeys {
+                cases = [.privacy, .contacts, .keys, .atteseter, .notifications, .legal, .experimental]
+            } else {
+                cases = allCases
+            }
+            return cases
+        }
     }
 
     private let decorator: SettingsViewDecoratorType
     private let currentUser: User?
+    private let organisationalRules: OrganisationalRules?
+    private let rows: [Settings]
 
     init(
         decorator: SettingsViewDecoratorType = SettingsViewDecorator(),
-        currentUser: User? = DataService.shared.currentUser
+        currentUser: User? = DataService.shared.currentUser,
+        organisationalRulesService: OrganisationalRulesServiceType = OrganisationalRulesService()
     ) {
         self.decorator = decorator
         self.currentUser = currentUser
+        self.organisationalRules = organisationalRulesService.getSavedOrganisationalRulesForCurrentUser()
+        self.rows = Settings.allCases(with: self.organisationalRules)
```

Comment:
The method should exactly be called `Settings.filtered` I think, and actually after my above suggestion, `SettingsMenuItem.filtered` maybe?

Reply:
done